src/client/features/extension-files.ts

```diff
@@ -0,0 +1,21 @@
+import { ClientCapabilities } from '../../protocol'
+import { XContentRequest, XFilesRequest } from '../../protocol/extension-files'
+import { _RemoteWorkspace } from '../../server/features/workspace'
+import { Client } from '../client'
+import { StaticFeature } from './common'
+
+export class FilesExtensionFeature implements StaticFeature {
```

Comment:
Remove "extension" because this is not an extension (it was an LSP extension but we'll just make it a core part of CXP, although not all clients need to implement it).